### DISCUSSION TITLE
[Support] Manage the remote 4G router through pivpn

### DISCUSSION THREAD

**Discussion by githubbuc** [pivpn/pivpn#1381] [Support]
Hi all
Do you know the way how i can manage the remote 4G router through a vpn?
If I connect to the 4G router lan, I manage the vpn server side router (router server vpn 192.168.1.1) without any problems. If, on the other hand, I connect to the LAN of the router on the vpn server side, I cannot manage the 4G router of the vpn client (client vpn router 191.168.0.1).

On both sides I have:
SERVER
pi01 		192.168.1.254
router_X	        192.168.1.1

CLIENT
pi02 		192.168.0.51
router_Y	        192.168.0.1

[pi01] ---ping---> [pi02]
[pi01] <---ping--- [pi02]

[pi01]  ---**NOTping**---> [roter 192.168.0.51]-[pi02]
[pi01]-[roter 192.168.1.254]  <---ping--- [pi02]

Thanks

**Comment by TinCanTech** [comment]
Reference: https://forums.openvpn.net/viewtopic.php?f=1&t=32731

@4s3ti @orazioedoardo - I had a look but I don't think `pivpn` has options for exposing the client-side LAN, can you please confirm _if_ that is the case ?

**Comment by coolapso** [comment]
Hmmm ... Nope i don't think we have that option, last time i remember exposing client side LAN involved a bit more tweaking outside of pivpn ... not sure if that changed in the meantime.

**Comment by TinCanTech** [comment]
Thanks for confirming and there is no change on the Openvpn side, it is still _complicated_

**Comment by githubbuc** [comment]
I have done what is written in the guidelines **_"Including multiple machines on the client side when using a routed VPN (dev tun)"_** and then I was able to ping client 192.168.0.51 from server. I have not encountered any problems guys! My problem now is that i cannot access the ip address of the 4G router througth vpn. I am not understanding why from the client LAN instead I connect to the ip address of the router server 192.168.1.1 and manage it.
